examples/siro_sandbox/test_gfx_replay.py

```diff
@@ -0,0 +1,112 @@
+import gzip
```

Comment:
Draft file that reads recorded keyframes and generates a video from observations.